backend/src/routes/api/status/statusUtils.ts

```diff
@@ -35,6 +36,13 @@ export const status = async (
       const adminUsers = await getGroup(customObjectsApi, adminGroup);
       isAdmin = adminUsers?.includes(userName) ?? false;
     }
+
+    if (allowedGroup === '') {
+      throw new Error('It is not allowed to set an empty string as allowed group.');
+    } else {
+      const allowedUsers = await getGroup(customObjectsApi, allowedGroup);
+      isAllowed = allowedUsers?.includes(userName) ?? false;
```

Comment:
There is a k8s feature that allows you to prefix the user in a group to allow for non-safe k8s name characters

I implemented something like this for the `adminUsers`.
https://github.com/opendatahub-io/odh-dashboard/blob/main/backend/src/routes/api/status/statusUtils.ts#L41

Reply:
Applied to both as part of merging the admin and allowed function